web/src/components/Required.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+
+const color = "#e11d48";
```

Comment:
Let's change it, but I would not use the `primary` variable but rather some dedicated yellow color (to keep it semantically different, since we are going here for "warning" and not "primary" meaning)